backend/src/test/resources/sequences_multi_segment.fasta

```diff
@@ -1,31 +1,31 @@
->custom1_main
+>custom1_notOnlySegment
```

Comment:
(I haven't got my head around this name - you can explain it to me :) )

Reply:
That was already used in the tests 😅  - I guess it's the segment name for the segment that is "not (the) only segment" 😅